Baseball/api/game.py

```diff
@@ -66,11 +66,13 @@ def bat(self, batter, pitcher, inning):
         Bat.objects.create(player=batter, inning=inning, result=result)
         return result
 
-    def at_bat(self, batter, pitcher):
+    def at_bat(self, batter, pitcher, inning):
         if self.hit(batter, pitcher):
             result = self.type_of_hit(batter, pitcher)
+            Bat.objects.create(player=batter, inning=inning, result=result)
         else:
             result = self.BATTING_RESULT_DICT['strike_out']
+            Bat.objects.create(player=batter, inning=inning, result=result)
```

Comment:
you can put the create Bat outside of the if statement and not have to repeat it